### ISSUE TITLE
Clone could not be found

### ISSUE THREAD

**Issue by AstroSmoke** [rlancaste/AstroPi3#2]
Hi, thank you for the main script, I tried the manual process with you guide and failed as sections could not be installed for various reasons, mainly newbie to Pi's and ubuntu installations.

Anyway the automatic method, worked except for the main clone download as it could not be found via the terminal window. 
git clone https://github.com/rlancaste/AstroPi3.git

I created a user at github and downloaded the clone zip manually and transferred it to the pi via a usb drive.
From there the install appears to have loaded so I can test it out.

Notes:
I am on the 2nd attempt, ran into problems with vnc, firefox and various other issues so reformatted the SD card and started again.
I am using a Pi2
Transcend 16gb SDcard
I first installed ubuntu mate, then ran the update and then the upgrade  
Then proceeded with your auto script guide. 
As noted above the clone seems to be the only problem.

I'm looking at trying out a Pi3B+, or Odroid XU4 or there is a new DFRpanda M series board as I would like to be able to do video planet and near field and unidentified objects.

**Comment by rlancaste** [comment]
If your computer cannot find git, then it needs to be installed before cloning.
If the clone command fails, that could be an issue with GitHub.
Was this resolved?

**Comment by rlancaste** [comment]
As to the Pi3B+, they still haven't released ubuntu for that yet.  I have been using an image that has been modified to work on one.  The Odriod machine can use my 64 bit script, I wrote it specifically for an Odroid C2 and it should also work on the XU4.

**Comment by rlancaste** [comment]
Sorry it took so long for me to respond.  I must have missed the message in my mailbox.

**Comment by rlancaste** [comment]
did  your issues get resolved?

**Comment by AstroSmoke** [comment]
HI, I shelved it for the moment. As a newbie to Astro imaging I find myself needing the crutch of direct connection and more importantly found I needed to add another component before I went remote control, the focuser. So I am building the Diyfocuserpro2 project as well as helping the project with some engineering load testing of stepper motors. In the long term I aim to be more remote to the scope, when I get more confident with everything I'll re-investigate software and hardware options.
